jq255.md

```diff
@@ -0,0 +1,1412 @@
+# The jq255e and jq255s Groups, Key Exchange and Signatures
+
+  - Version: 0.0.0
+  - Author: Thomas Pornin
+
+## Introduction
+
+This document specifies the **jq255e** and **jq255s** groups. These are
+prime order groups based on elliptic curves, with characteristics
+suitable for building cryptographic protocols:
+
+  - The order of each group is a prime integer close to `2^254`, hence
+    formally offering "127-bit security" against discrete logarithm
+    attacks.
+
+  - Each group element (including the neutral element) can be encoded as
```

Comment:
I think we call the neutral element the identity in Ristretto and I've been trying to use that across groups. Opinions?

Reply:
"Identity" works. In French we usually say _élément neutre_, hence my use of "neutral" by default, but this is certainly arguable and no big deal.

Reply:
I renamed the neutral into "identity".